custom/panel.libsonnet

```diff
@@ -32,14 +32,18 @@ local getPanelName(type) =
     // interesting from a reusability standpoint.
     + self.queryOptions.withDatasource('datasource', '-- Mixed --'),
 
-  link+: { '#':: d.package.newSub('link', '') },
-  thresholdStep+: { '#':: d.package.newSub('thresholdStep', '') },
-  transformation+: { '#':: d.package.newSub('transformation', '') },
-  valueMapping+: { '#':: d.package.newSub('valueMapping', '') },
+  // Backwards compatible entries, ignored in docs
+  link+: self.panelOptions.link + { '#':: { ignore: true } },
+  thresholdStep+: self.standardOptions.threshold.step + { '#':: { ignore: true } },
+  transformation+: self.queryOptions.transformation + { '#':: { ignore: true } },
+  valueMapping+: self.standardOptions.mapping + { '#':: { ignore: true } },
+  fieldOverride+: self.standardOptions.override + { '#':: { ignore: true } },
```

Comment:
These have been moved into the separate sections so that they align nicely with the newly generated subpackages, keeping these here for backwards compatibility.